tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/ESEmailAddressContactSearchEngine.java

```diff
@@ -0,0 +1,19 @@
+package com.linagora.tmail.james.jmap.model;
+
+import org.apache.james.jmap.api.model.AccountId;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Mono;
+import scala.runtime.BoxedUnit;
+
+public class ESEmailAddressContactSearchEngine implements EmailAddressContactSearchEngine{
+    @Override
+    public Publisher<BoxedUnit> index(AccountId accountId, EmailAddressContact contact) {
+        //Mono.fromCallable()
```

Comment:
I think you can reuse ElasticSearchIndexer from the backends?

Reply:
Yup, again I haven't finished it yet :(

Reply:
I know you didn't finish, it's just early feedback. It's not here to put a hammer above your head, just give you some guidance and helping you keeping track of what is left :)